frontend/app/component/form/candidates_votes_form/CandidatesVotesForm.tsx

```diff
@@ -31,79 +39,93 @@ export interface CandidatesVotesFormProps {
 export function CandidatesVotesForm({ group }: CandidatesVotesFormProps) {
   const { register, format, deformat, warnings: inputMaskWarnings } = usePositiveNumberInputMask();
   const formRef = React.useRef<CandidatesVotesFormElement>(null);
-  const {
-    sectionValues,
-    errors,
-    warnings,
-    setSectionValues,
-    loading,
-    serverError,
-    isCalled,
-    setTemporaryCache,
-  } = usePoliticalGroup(group.number);
 
-  usePreventFormEnterSubmit(formRef);
+  const _IGNORE_WARNINGS_ID = `candidates_votes_form_ignore_warnings_${group.number}`;
 
-  const getValues = React.useCallback(
-    (elements: CandidatesVotesFormElement["elements"]): PoliticalGroupVotes => {
-      const candidate_votes: CandidateVotes[] = [];
-      for (const el of elements["candidatevotes[]"]) {
-        candidate_votes.push({
-          number: candidateNumberFromId(el.id),
-          votes: deformat(el.value),
-        });
-      }
+  const getValues = React.useCallback(() => {
+    const form = formRef.current;
+    if (!form) {
       return {
         number: group.number,
-        total: deformat(elements.total.value),
-        candidate_votes: candidate_votes,
+        total: 0,
+        candidate_votes: [],
       };
-    },
-    [deformat, group],
-  );
+    }
 
-  const errorsAndWarnings = useErrorsAndWarnings(errors, warnings, inputMaskWarnings);
-
-  //const blocker = useBlocker() use const blocker to render confirmation UI.
-  useBlocker(() => {
-    if (formRef.current && !isCalled) {
-      const elements = formRef.current.elements;
-      const values = getValues(elements);
-      setTemporaryCache({
-        key: "political_group_votes",
-        id: group.number,
-        data: values,
+    const elements = form.elements;
+    const candidate_votes: CandidateVotes[] = [];
+    for (const el of elements["candidatevotes[]"]) {
+      candidate_votes.push({
+        number: candidateNumberFromId(el.id),
+        votes: deformat(el.value),
       });
     }
+    return {
+      number: group.number,
+      total: deformat(elements.total.value),
+      candidate_votes: candidate_votes,
+    };
+  }, [deformat, group]);
+
+  const getIgnoreWarnings = React.useCallback(() => {
+    const checkbox = document.getElementById(_IGNORE_WARNINGS_ID) as HTMLInputElement | null;
```

Comment:
Why not use a React `ref` for this? (And same for the checkbox in other form components)

Reply:
this is one of things I would like to change, store all values and changes directly in a ref type object centrally. 
reduces complexity and test timing failures.

in this case, ref was a bit flaky when called from the PollingStationNavigation

Reply:
Maybe because the value can be null, but `getElementById` has the same issue.. so not sure why a ref is more flaky than this. We can look at it later though.

Reply:
I will create an issue for this.

Reply:
#290